sdcm/sct_runner.py

```diff
@@ -1349,3 +1352,8 @@ def clean_sct_runners(test_status: str,
             end_message = "No runners have been terminated"
 
     LOGGER.info(end_message)
+
+
+class AwsFipsSctRunner(AwsSctRunner):
+    VERSION = f"{SctRunner.VERSION}-fips"
+    BASE_IMAGE = 'resolve:ssm:/aws/service/marketplace/prod-k6fgbnayirmrc/latest'
```

Comment:
How do we know that when we start this image the fips is enabled?
Is there a way to check that in the test maybe?

Reply:
We have that in the fips artifact test.

I need to check when it's done, I think if we identify it's Ubuntu Pro, when check it

Anyhow it's the official product from the Ubuntu title as fips, and so far we took a specific AMI out then, this syntax enables us to get the latest each time.

Anyhow we aren't gonna build those too often

Reply:
One more "proof" for now,  that it reproduces the crash, that is being worked on. (Not sure by whom, that might would want to try it out as part of investigations and fixes)

Reply:
here is the check
https://github.com/scylladb/scylla-cluster-tests/commit/64b4e506ff4a17e7bf9d6ffe9c04242b88162460

Reply:
but when I think of it, it's relevant to the db nodes
no to the sct-runner or the builder